tests/js/setup.ts

```diff
@@ -115,7 +116,11 @@ jest.mock('@sentry/react', function sentryReact() {
     captureMessage: jest.fn(),
     captureException: jest.fn(),
     showReportDialog: jest.fn(),
-    startSpan: jest.fn(),
+    startSpan: jest
+      .fn()
+      .mockImplementation((context, callback) =>
+        callback(new SentryReactCore.Span(context))
```

Comment:
this should be exported from `@sentry/react`, can we do `jest.spyOn(SentryReact, 'startSpan'),` as the mock?

Reply:
Seems to work fine! If you're fine with it, I am, too. I originally avoided it because all the other functions in the mocked module are fully stubbed out, and I didn't know what side effects `startSpan` has, so I felt it's safer to isolate to something more primitive like the `Span` constructor

Reply:
https://github.com/getsentry/sentry/pull/60525/commits/66224ec59af7851bb2a4e3321f5a6c0f94fd8546